src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, T value);
```

Comment:
Ensure that the size of the list passed to the `fill()` method remains the same after the method execution. Replace old values with new ones, don't add or remove elements from the list.